HISTORY.md

```diff
@@ -2,10 +2,17 @@
 
 ## in development:
 
+## v6.9.1
+This is a small patch to support projects which have begun to use v7 development.
+
+* **Notable Changes:**
+  * Snap! now offers to redirect you to the development version if you load a file with a newer serializer version.
+  * Extension authors who maintain development versions may want to set `IDE_Morph.prototype.developmentPath`
+  * Extension authors who want to disable this behavior should set `IDE_Morph.prototype.developmentPath = 'snap.html'`
 ## 6.9.0
 * **Notable Changes:**
     * JS-functions are now disabled by default until switched on in the settings menu per session
-    * error messages in presentation mode are now shown as pop-up messages onstage  
```

Comment:
Kinda annoying, but all the changes below in this file are just trimming whitespace.